More important docs! (#2317)
spacedriveapp/spacedrive: docs/product/guides/importing-photos.mdx

---
index: 20
---

# Importing Photos

> Spacedrive is designed to be a central location for all your media, with [Media View](), viewing and managing your photos easy, with a variety of filters and sorting options just like traditional photo apps, so to compliment this we have (and are working on) a variety of ways to import your photos.

Reviewer: This segment in general is a bit weird, I think, and so we might be better off rephrasing it to:
```suggestion
Spacedrive is designed to be a central location for all your media. The [Media View]() makes viewing and managing your photos easy with a variety of filtering and sorting options. To compliment this we also have (and are working on) a variety of ways to import your photos.
```

---
index: 20
---

# Importing Photos

> Spacedrive is designed to be a central location for all your media. This means you can import photos from a variety of sources, including your local hard drive, Apple Photos, Google Photos and more.

Author: local storage could be confused with localStorage, no?

> Spacedrive is designed to be a central location for all your media. This means you can import photos from a variety of sources, including your local hard drive, Apple Photos, Google Photos and more.

Reviewer: Hm, not sure what you're referring to by `localStorage` in this context, the Web API? I suppose that _might_ be a concern, but as this is a user-facing guide I think most people would understand the meaning of "local storage" in this context, perhaps even better than "hard drives", to refer to data stored locally on their machine.

I mostly meant to move away from "hard drive" as that usually refers to mechanical drives, which are less and less common nowadays. I suppose simply "drive" could be used instead too but that might also conflict with Google Drive.

Decisions, decisions...

> Spacedrive is designed to be a central location for all your media. This means you can import photos from a variety of sources, including your local hard drive, Apple Photos, Google Photos and more.

Author: Turns out while merging I already replaced this paragraph anyway, I agree on the term "hard drive", very dated and somewhat incorrect.

## From an SD card

> Currently you can add the SD card as a location and copy the files to a specific folder, in an upcoming release we will ship a dedicated interface for importing media, with option to exclude already imported media. It will also use your preference for photo storage and place the new media in the designated folder.

Reviewer: ```suggestion
Currently you can add the SD card as a location and copy the files to a specific folder. In an upcoming release we will ship a dedicated interface for importing media, with option to exclude already imported media. It will also use your preference for photo storage and place the new media in the designated folder.
```
